src/compas_view2/objects/object.py

```diff
@@ -34,3 +64,50 @@ def create(self):
 
     def edit(self):
         pass
+
+    @property
+    def translation(self):
+        return self._translation
+
+    @translation.setter
+    def translation(self, data):
+        self._translation = data
+
+    @property
+    def rotation(self):
+        return self._rotation
+
+    @rotation.setter
+    def rotation(self, data):
+        self._rotation = data
+
+    @property
+    def scale(self):
+        return self._scale
+
+    @scale.setter
+    def scale(self, data):
+        self._scale = data
```

Comment:
why use setters and getters if we don't process the input?

Reply:
A mistake of mine.... was intended to do 
`self._translation.data = data`
updating now

Reply:
don't think that makes sense. just assign the object, no? or if necessary the matrix, but that also seems redundant...